server/api/teams/index.get.ts

```diff
@@ -29,4 +31,7 @@ export default defineEventHandler(async (): Promise<TeamsResponse | H3Error> =>
       message: (error as Error).message
     })
   }
+  finally {
```

Comment:
as both `try` and `catch` clauses have returns, the `finally` block will never occur - also the connection will already be destroyed

Reply:
meh, you are tight, sorry